### ISSUE TITLE
Retitle

### ISSUE THREAD

**Issue by ariel1325** [steam3d/MagicPods-Windows#72]
Hey, can you please enable an option to rename the headphones? like : "magic" AirPods
Thank you

**Comment by steam3d** [comment]
Hi. You can use any name, except  using emoji. 
To do:
1. Close the MagicPods
2. Go to Devices and printers -> Select your AirPods -> Properties -> Bluetooth 
3. Rename your AirPods as you wish. (Do not use emoji. Windows not supports them)
4. Run the MagicPods

![image](https://user-images.githubusercontent.com/30986874/107331416-ea7a9080-6ac3-11eb-8888-54d11ff961fc.png)

**Comment by ariel1325** [comment]
Working! thank you so much

**Comment by surfaceflinger** [comment]
On Apple devices it's possible to rename AirPods and then other devices can see the changed name. Would it be possible to reimplement this in MagicPods? Renaming names using Windows bluetooth settings changes the name only visually, on the same pc.

**Comment by steam3d** [comment]
The Windows save any headphones name new only  on the first pairing process. Apple device check every time the name after you connect headphones (i think). I did not found any API to change this behavior for Windows.

For example if you have 2 computers with paired headphones. On first one you renamed headphones as "My headphones", to push changes to the second computer you must unpair and then pair headphones to computer and name will be updated.

I wrote your wish, but i think it almost impossible, because it easy to do for specific Bluetooth adapter, but Windows supports hundreds of thousands of different Bluetooth adapters
